app/controllers/restaurants_controller.rb

```diff
@@ -2,47 +2,41 @@ class RestaurantsController < ApplicationController
   skip_before_action :verify_authenticity_token
 
   def index
-    Restaurant.sorted
+    render json: { restaurants: Restaurant.sorted_by_name.map{|r| RestaurantSerializer.new(r)} }
```

Comment:
Not sure, but you might be able to use `each_serializer` instead of instantiating it again and again for each restaurant, check: https://stackoverflow.com/questions/36278413/serialize-an-array-of-models-using-active-model-serializers
(Didn't read all the way through so sorry if that's not possible)

Reply:
I tried to use it firstly, but the array of the serialized restaurants wasn't wrapped by the "restaurants" key. I couldn't find another way of doing so.